src/app/public/modules/repeater/repeater-item.component.ts

```diff
@@ -184,6 +211,58 @@ export class SkyRepeaterItemComponent implements OnDestroy, OnInit {
     this.inlineFormClose.emit(inlineFormCloseArgs);
   }
 
+  public moveToTop(event: Event): void {
+    event.stopPropagation();
+    this.adapterService.moveItemUp(this.elementRef, true);
+  }
+
+  public handleKeyboardEvent(event: KeyboardEvent): void {
+    let key = event.key.toLowerCase();
+    if (key === ' ') {
```

Comment:
Would this be a little more rigid if we used keycodes (`32`)?

Reply:
Potentially but `keycode` is considered deprecated. `key` does us a standard value across browsers according to MDN.

https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode
https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key

Reply:
Ok that works. This is also news to me. I'm now concerned about other components we have where we use keycodes. 😲